atomic_reactor/plugins/pre_inject_yum_repo.py

```diff
@@ -129,6 +129,9 @@ def run(self):
         """
         run the plugin
         """
+        if base_image_is_scratch(self.workflow.builder.base_image.to_str()):
+            self.log.debug("base image is scratch, do not inject yum repo")
+            return
```

Comment:
Isn't there yum commands run in this plugin? Which is not possible on a blank base image.